tlatools/org.lamport.tlatools/javacc/tla+.jj

```diff
@@ -3413,28 +3039,6 @@ ParenthesesExpression() : {
   {  return tn; }
 }
 
-SyntaxTreeNode
-ClosedExpressionOrOp() : {
```

Comment:
This production is mentioned in a big comment up top:

    check ClosedExpressionOrOp [...] ClosedExpressionOrOp is irrelevant here

as well as another:

    If a ClosedExpressionOrOp node is found when parsing an expression, [...]

I'm not entirely sure how those comments should be updated, but they need attention.

Reply:
I rewrote the first comment and looking at the second comment & the generated code, it turns out that anchor element is totally unused (value set but never read) so I just deleted it.